lib/decidim/verifications/omniauth/bosa_action_authorizer.rb

```diff
@@ -12,7 +12,7 @@ def authorize
 
           if status_code == :ok && action == "vote" && minimum_age.present? && under_minimum_age?(minimum_age)
             status_code = :unauthorized
-            data[:fields] = { "date_of_birth" => I18n.t("decidim.verifications.omniauth.errors.minimum_age", minimum_age: minimum_age, locale: authorization.user.locale) }
+            data[:fields] = { "date_of_birth" => I18n.t("decidim.verifications.omniauth.errors.minimum_age", minimum_age: minimum_age, locale: current_locale) }
```

Comment:
Sorry my review was to quick.
current_locale is not available in this context.
That's why it is based on the local received from the authorization

Reply:
Or we could use I18n.locale